### PR TITLE
Freemabd1/complete schema - Add some more bigquery test cases and made sure the json schema files contain all the columns.

### PR DESCRIPTION
**Pull request by freemabd1**
Add some more bigquery test cases and made sure the json schema files contain all the columns.

### CODE REVIEW

File: api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java [hunk 1]
```diff
@@ -30,7 +31,13 @@
 
     @Override
     public List<String> getTableNames() {
-        return Arrays.asList("icd9_criteria", "person", "CONCEPT", "condition_occurrence");
+        return Arrays.asList(
+                "icd9_criteria",
+                "person",
+                "CONCEPT",
```
Comment (danqrodney) [inline api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java]: Is there a reason this one is in all caps?

File: api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java [hunk 2]
```diff
@@ -42,33 +49,97 @@ public void getCriteriaByTypeAndParentId() throws Exception {
...
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
+
+        SubjectListResponse listResponse = (SubjectListResponse) response.getBody();
+        String subject = listResponse.get(0);
+        assertThat(subject).isEqualTo("1,1,1");
+    }
+
+    @Test
+    public void searchSubjects_ProcedureOccurrenceLeaf() throws Exception {
+        ResponseEntity response = controller
```
Comment (danqrodney) [inline api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java]: You can make all these test cases shorter with a helper function... like:

assertSubjects(createSearchRequest("...."), "2,2,2", ....)

File: api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java [hunk 3]
```diff
@@ -122,64 +135,58 @@ public void findParametersWithEmptyDomainIds() throws Exception {
 
     @Test
     public void getSubQuery() throws Exception {
+        String TABLE_PREFIX = workbenchConfig.bigquery.projectId + "." + workbenchConfig.bigquery.dataSetId;
```
Comment (danqrodney) [inline api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java]: Nit: only constants should be in all caps. You could probably have a getTablePrefix() helper method that returns this, since it's repeated.

File: api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java [hunk 4]
```diff
@@ -122,64 +135,58 @@ public void findParametersWithEmptyDomainIds() throws Exception {
...
         Map<String, String> expectedByKey = new HashMap<String, String>();
         expectedByKey.put("Condition",
             "SELECT DISTINCT PERSON_ID " +
```
Comment (danqrodney) [inline api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java]: Nit: make all table names and column names lower case? 

Making everything all caps makes it hard to parse.

File: api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java (hunk 1 above)
Comment (johnstill) [reply api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java]: BigQuery table names are case sensitive, ref:
https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#case-sensitivity.  To make this lowercase we'll have to adjust the table schema in BQ.

File: api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java (hunk 4 above)
Comment (johnstill) [reply api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java]: https://github.com/all-of-us/workbench/pull/159#discussion_r141481302

Column names are actually *not* case sensitive (unlike table names), so we _could_ make them all lowercase, but the table names would still need addressing in the db itself.

File: api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java (hunk 1 above)
Comment (freemabd1) [reply api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java]: Yeah.. table names are case sensitive in BigQuery.

Comment (danqrodney) [reply api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java]: OK. Let's file an issue to change the table names in BQ to be all lower-case?

Comment (freemabd1) [reply api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java]: Yep.. should that be posted in the RW project in jira or somewhere esle? And who should I assign it to?

Comment (danqrodney) [reply api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java]: RW project is fine. You can assign it to me if you want.
